.gitignore

```diff
@@ -1 +1 @@
-proposals/**/index.html
+proposals/*/index.html
```

Comment:
was that "**" intentional? was it intended to reach e.g. `proposals/foo/bar/index.html` ?